<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools</class>
 <widget class="QWidget" name="Tools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="toolBorder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>194</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(38, 39, 40);
border-style: solid;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mouseButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="brushButton">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="circleButton">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>310</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="rectangleButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="eraserButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="gridButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="overlayButton">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>370</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="currentColorButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(120, 120, 120); 
border-style: solid; 
border-width: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="togglePreviewButton">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>250</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="brushSize">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>170</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="fpsSpeed">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>439</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fpsSpeedLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>440</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="numberOfFrames">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>470</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="numberOfFramesLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>470</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Frames</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="colorBorder">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>19</y>
     <width>94</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 39, 40);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>toolBorder</zorder>
  <zorder>fpsSpeed</zorder>
  <zorder>fpsSpeedLabel</zorder>
  <zorder>numberOfFrames</zorder>
  <zorder>numberOfFramesLabel</zorder>
  <zorder>colorBorder</zorder>
  <zorder>mouseButton</zorder>
  <zorder>brushButton</zorder>
  <zorder>circleButton</zorder>
  <zorder>rectangleButton</zorder>
  <zorder>eraserButton</zorder>
  <zorder>gridButton</zorder>
  <zorder>overlayButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>currentColorButton</zorder>
  <zorder>togglePreviewButton</zorder>
  <zorder>brushSize</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
